airbyte-integrations/connectors/destination-local-json/src/main/java/io/airbyte/integrations/destination/local_json/LocalJsonDestination.java

```diff
@@ -152,6 +152,11 @@ public JsonConsumer(Map<String, WriteConfig> writeConfigs, ConfiguredAirbyteCata
       this.writeConfigs = writeConfigs;
     }
 
+    @Override
```

Comment:
You could have an empty implementation on the base class.

Reply:
I think the goal is to get destinations to use the start functionality actually. I didn't do it in all cases because I didn't want to increase scope of this PR even more. But I think in the end we don't expect any of these to be no ops. In practice this is just moving the code out of the write method into the start method in many cases.